apps/condo/domains/common/utils/phone.js

```diff
@@ -1,8 +1,12 @@
 const { phone } = require('phone')
 
+
+const STRICT_PHONE_REGEX = /^\+[0-9\s()-]+$/
```

Comment:
What about spaces?

Reply:
what do you mean? Spaces are used for formatting, so we allow them, there are tests for cases with spaces